### PR TITLE
Add zmsort for tgicl

### PR DESCRIPTION
**Pull request by noirot**
Y

* [X ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/zmsort/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package:
+  name: zmsort
+  version: "1.0"
+
+source:
+  fn: zmsort.tar.gz
+  url: ftp://occams.dfci.harvard.edu/pub/bio/tgi/software/tgicl/zmsort.tar.gz
+  md5: XXXX
```
Comment (bgruening) [inline recipes/zmsort/meta.yaml]: The md5 checksum should be `33e54b4f9b7956c307a2529576f51b10`

File: recipes/zmsort/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+   
+requirements:
+  build:
+    - gcc   # [not osx]
```
Comment (bgruening) [inline recipes/zmsort/meta.yaml]: `- llvm  # [osx]` is missing here.

File: recipes/zmsort/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  run:
+    - libgcc    # [not osx]
+
+test:
+  commands:
+    - zmsort zmsort
+
+
+about:
+  home: ftp://occams.dfci.harvard.edu/pub/bio/tgi/software/tgicl/zmsort.tar.gz
```
Comment (bgruening) [inline recipes/zmsort/meta.yaml]: This needs to be the homepage not the tarball :)

**Comment by noirot** [comment]
@bgruening , I'm waiting for long time, do I have to do something for running checks ?

**Comment by bgruening** [comment]
@noirot if many people create PR this testing can take some long time. Sorry about that.

File: recipes/zmsort/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  build:
+    - gcc   # [not osx]
+    - gclib 
+    - llvm # [osx]
+  run:
+    - libgcc    # [not osx]
+
+test:
+  commands:
+    - zmsort README
```
Comment (bgruening) [inline recipes/zmsort/meta.yaml]: README does not exists in the test/runtime environment. Can we do the same `grep` trick as with the other recipe?

**Comment by johanneskoester** [comment]
We  have made changes to our build system (migrated from Travis CI to  Circle CI). If you would like to continue working on this PR, please  reopen this PR and update with latest upstream/master.

**Pull request closed without merging**